app/src/main/java/com/example/flexioffice/data/UserRepository.kt

```
@@ -77,6 +77,25 @@ class UserRepository
+                    firestore
+                        .collection(USERS_COLLECTION)
+                        .document(uid)
+                        .get()
+                        .await()
+
+                if (document.exists()) {
+                    document.toObject(User::class.java)?.copy(id = document.id)
+                } else {
+                    null
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)